schematics/ng-add/dependencies.ts

```diff
@@ -33,16 +33,13 @@ export function getSingleSpaAngularDependency(): NodeDependency {
   };
 }
 
-/**
- * We have to install `@angular-builders/custom-webpack` version compatible with the current
- * version of Angular. If Angular is 8 then `custom-webpack@8.4.1` has to be installed.
- */
 export function getAngularBuildersCustomWebpackDependency(): NodeDependency {
-  const { VERSION } = require('@angular/core');
-
   return {
     name: '@angular-builders/custom-webpack',
-    version: `^${VERSION.major}`,
+    // Relying on the current major version of Angular is wrong.
+    // We have to install the `latest` version because we cannot know
+    // when the new version will be published.
+    version: 'latest',
```

Comment:
Does the latest version of `@angular-builders/custom-webpack` work with Angular 9 and 10?

Reply:
You're right. Do you think it makes sense to check whether the version exists using the NPM API?

Reply:
> Do you think it makes sense to check whether the version exists using the NPM API?

Yeah that could work. Alternatively, maybe we maintain a map of versions?

```js
const angularVersionToCustomWebpackVersion = {
  9: 9,
  10: 10,
  11: 10
}
```

I'm open to whichever approach is most accurate and maintainable

Reply:
Umm, I'm just afraid that we will need to change that map and publish new versions.

Reply:
Agreed - that would be unfortunate. So you're suggesting using the same major version if it exists, and falling back to `latest` otherwise? I'm okay with that, and agree it will require less maintenance.

Reply:
Yeah. Just wondering how to check if the `^10` or `^11` version exists.

Reply:
Here's one way I've done it in the past: https://github.com/esm-bundle/react-dom/blob/4c485bb1f7364b870ee9ec5c09a0f966243d40a0/rollup.config.js#L57.

Reply:
Another option would be to use [`npm view`](https://docs.npmjs.com/cli/v6/commands/npm-view) which we use in [create-single-spa to check if there are updates to it.](https://github.com/single-spa/create-single-spa/blob/29c27038478762430285fd289ad8372c3d0bc2ea/packages/generator-single-spa/src/generator-single-spa.js#L39)